README.md

```diff
@@ -65,8 +65,10 @@ Build with ❤️️ from 🇫🇷 🇯🇵 🇬🇷 🇸🇪 🇺🇸
 
 ---
 
-## Collecting users feedback
+## Helping you get started
+
+We have 30 min slots for 1:1 [tech discussions and overviews](https://calendly.com/stephanejourdan/30min) twice a day to help you get started and collect users feedback.
 
-Have you launched driftctl ? Please take a moment to answer the [following questions](https://driftctl.typeform.com/to/fYUOOMj3) and help us improve the tool. 
+:warning: ** No sales involved **: You will deal with a technical team member of the project. 
```

Comment:
Remove whitespaces if you wanna a bold text `**No sales involved**`

Reply:
removing the whole sentence